database/schema.sql

```diff
@@ -0,0 +1,36 @@
+create database yump_SF
+
+use yump_SF
+
+create table restaurant
+(
+  id   int auto_increment
+    primary key,
+  name text null
```

Comment:
wouldn't it be better to have name as NOT NULL? Having NOT NULL would ensure you that you cannot insert an instance without a name of the restaurant.